packages/mc-html-template/lib/compile-html.js

```diff
@@ -0,0 +1,66 @@
+const fs = require('fs');
+const path = require('path');
+const fetch = require('node-fetch');
+const loadEnv = require('./load-env');
+const loadHeaders = require('./load-headers');
+const replaceHtmlPlaceholders = require('./utils/replace-html-placeholders');
+
+const requiredOptions = ['envPath', 'publicAssetsPath'];
+
+/**
+ * Options:
+ * - envPath
+ * - cspPath
+ * - publicAssetsPath
+ * - useLocalAssets
+ */
+module.exports = async function compileHtml(options) {
```

Comment:
Most of the logic here has been extracted from the `mc-http-server/bin/start-mc.js`